idelib/dataset.py

```diff
@@ -1103,7 +1110,10 @@ def __eq__(self, other):
                and self.displayRange == other.displayRange \
                and self.hasDisplayRange == other.hasDisplayRange \
                and self.sessions == other.sessions \
-               and self.allowMeanRemoval == other.allowMeanRemoval  
+               and self.allowMeanRemoval == other.allowMeanRemoval
+
+    def _initializeCache(self, *args, **kwargs):
+        raise NotImplemented("EventArrays don't support caching")
```

Comment:
Since this isn't a method called by a user, would this be better as a warning?

Reply:
I should probably just have it refer up to the parent instead of raising an error.  I'm going to be reworking the caching soon-ish so it may become moot.